Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -130,5 +156,59 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
     return self
   }
 % end
+  
+  public func lineComment(_ text: String) -> Self {
+    
+    let textWithSlashesPieces = prefixLines(of: text, with: "//")
+      .map { TriviaPiece.lineComment("\($0)") }
+    return addCommentPiece(commentPieces: textWithSlashesPieces)
+  }
+  
+  /// A documentation block comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text:  inserted between /*  and  */.
+  /// - Returns: a `${type.buildable()}` with the added line comment.
+  public func blockComment(_ text: String) -> Self {
+    let piece = TriviaPiece.blockComment("/* \(text) */")
+    return addCommentPiece(commentPieces: [piece])
+  }
+  
+  /// A documentation line comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text: comment to be inserted after ///
+  /// - Returns: a `${type.buildable()}` with the added documentation line comment.
+  public func docLineComment(_ text: String) -> Self {
+    let textWithSlashesPieces = prefixLines(of: text, with: "//")
+      .map { TriviaPiece.docLineComment("\($0)") }
+    return addCommentPiece(commentPieces: textWithSlashesPieces)
+  }
+
+  
+  /// A documentation block comment, starting with '/**' and ending with '*/'.
+  /// - Parameter text: comment to be inserted between /** and '*/.
+  /// - Returns: a `${type.buildable()}` with the added  documentation block comment.
+  public func docBlockComment(_ text: String) -> Self {
+    let piece = TriviaPiece.docBlockComment("/** \(text) */")
+    return addCommentPiece(commentPieces: [piece])
+  }
+  
+  private func addCommentPiece(commentPieces: [TriviaPiece]) -> Self {
```

Comment:
Per the [Swift Naming guidelines](https://www.swift.org/documentation/api-design-guidelines/#name-according-to-side-effects) this should be named `addingCommentPiece` because it returns a modified version of `self` instead of modifying `self` in-place.